support/Pipelines/Bbh/InitialData.py

```diff
@@ -393,6 +405,14 @@ def generate_id(
     ),
     help="Directory where steps in the pipeline are created.",
 )
+@click.option(
+    "--edit-readme/--no-edit-readme",
+    default=None,
+    help=(
+        "Edit the README file in the pipeline directory or run directory. "
+        "If not specified, a prompt will ask whether to edit the README."
```

Comment:
I'm not a huge fan of always being prompted for this. I'd like the option, but just not being forced to do it by default. I think this should just be a bool that defaults to false.

Reply:
Without the prompt there's little point to have the option. The idea is to encourage writing notes into a readme file.